.github/workflows/translations.yml

```diff
@@ -0,0 +1,106 @@
+name: Translations
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - .github/workflows/translations.yml
+      - frontend/**
+  pull_request:
+    paths:
+      - .github/workflows/translations.yml
+      - frontend/**
+
+jobs:
+  run-linters:
+    name: Check translations
+    runs-on: ubuntu-22.04
```

Comment:
Any reason not to use `ubuntu-latest`? We use everywhere else in the workflows except:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/fac8348489d0c5f4c840e2acc165a67ebbbb32c0/.github/workflows/e2e-tests.yml#L31

Reply:
Yeah, my thinking is that I think we're trying to avoid having floating version numbers to help us avoid nasty surprises.

Reply:
Worth creating an issue for changing the current instances of `ubuntu-latest`?

Reply:
Good plan.  I opened #4589.